### PR TITLE
Lint project source code

### PR DESCRIPTION
**Pull request by jugglinmike**

### CODE REVIEW

File: lib/metadata/filenames.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = function(issue) {
         .then(function(files) {
             return {
                 all: files.map(funk.prop('filename')),
-                ignoreRemoved: files.filter(function(file) { return file.status != "removed" }).map(funk.prop('filename'))
+                ignoreRemoved: files.filter(function(file) { return file.status != "removed"; }).map(funk.prop('filename'))
```
Comment (lukebjerring) [inline lib/metadata/filenames.js]:
```suggestion
                ignoreRemoved: files.filter(file => file.status != 'removed').map(funk.prop('filename'))
```

**Comment by jugglinmike** [comment]
All set, @lukebjerring!

**Comment by foolip** [comment]
Merged into master as a56b66951ab92ab1e4594f47d05bc97b8b7f20a1. Unfortunately I didn't have permission to push to the fork or I would have updated this PR directly.

@jugglinmike creating branches directly in this repo would make this sort of thing easier, even if the "allow contributors to modify" bit were set on the PR.

**Comment by jugglinmike** [comment]
We'll face the same friction for contributors who don't have "write" privileges. I'm happy to rebase whenever reviewers have bandwidth :)

**Comment by foolip** [comment]
The friction is for reviewers though, who will have write access. Anyway, not a big problem.

**Pull request closed without merging**